src/incubator/Dialog/index.tsx

```diff
@@ -24,10 +47,36 @@ const Dialog = (props: DialogProps) => {
     }
   }, [visible]);
 
-  return <ImperativeDialog {...others} initialVisibility={initialVisibility.current} ref={dialogRef}/>;
+  const style = useMemo(() => {
+    return [styles.defaultDialogStyle, containerStyle];
+  }, [containerStyle]);
+
+  return (
+    <ImperativeDialog {...others} initialVisibility={initialVisibility.current} ref={dialogRef}>
+      <View style={style}>
+        <DialogHeader {...headerProps}/>
+        {children}
+      </View>
+    </ImperativeDialog>
+  );
 };
 
 Dialog.displayName = 'Incubator.Dialog';
 Dialog.directions = DialogDirectionsEnum;
+Dialog.Header = DialogHeader;
```

Comment:
Why do you expose these components? Aren't they private and controlled by the 'dialogHeaderProps'?

Reply:
The idea is to give a default header but allow the user to override, so we give them (and possibly us in private) the option use our basic components to build something more complex.